allennlp/models/constituency_parser.py

```diff
@@ -61,7 +60,7 @@ class SpanConstituencyParser(Model):
     encoder : ``Seq2SeqEncoder``, required.
         The encoder that we will use in between embedding tokens and
         generating span representations.
-    feedforward_layer : ``FeedForward``, required.
```

Comment:
This will break existing models in the demo - you'll need to update it manually

Reply:
I'm pretty sure in every case I changed the constructor parameter to match what `from_params` had been expecting, so the models would keep working? (because they were all using from_params)

Reply:
Oh yep, my bad.